### ISSUE TITLE
v0.7.4 - error in debian/changelog

### ISSUE THREAD

**Issue by kreijack** [stefanberger/libtpms#182]
In the version 0.7.4, in the debian/changelog file there is an error which prevent the debian package building.
There is a misspell of the package name: li**tpms**s instead of li**btpm**s 
Below the diff to correct it

diff --git a/debian/changelog b/debian/changelog
index 5df4344..42549b8 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,4 +1,4 @@
-litpmss (0.7.4) RELEASED; urgency=medium
+libtpms (0.7.4) RELEASED; urgency=medium
 
   * Addressed potential constant-time issues in TPM 1.2 and TPM 2 code

**Comment by stefanberger** [comment]
This error is fixed in the stable-0.7.0 branch:
```
commit 984ad238dee7df879e755962bf880d6320636b08
Author: Stefan Berger <stefanb@linux.vnet.ibm.com>
Date:   Sun Nov 1 19:13:50 2020 -0500

    debian: Fix typo in changelog

    Signed-off-by: Stefan Berger <stefanb@linux.ibm.com>

diff --git a/debian/changelog b/debian/changelog
index 5df43445c..42549b830 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,4 +1,4 @@
-litpmss (0.7.4) RELEASED; urgency=medium
+libtpms (0.7.4) RELEASED; urgency=medium

   * Addressed potential constant-time issues in TPM 1.2 and TPM 2 code
```

**Comment by stefanberger** [comment]
I will probably create a v0.7.5 soon then this issue should be resolved.

**Comment by stefanberger** [comment]
I just tagged 0.7.5.

**Comment by stefanberger** [comment]
Is this issue now resolved with libtpms 0.7.5 or 0.8.0?

**Comment by kreijack** [comment]
Hi,
I just compiled and installed your package. So yes, this issue is solved for me.
Many thanks
